recipe/meta.yaml

```
@@ -1,53 +1,90 @@
 {% set name = "scikit-criteria" %}
-{% set version = "0.8.3" %}
+{% set version = "0.8.6" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-criteria-{{ version }}.tar.gz
-  sha256: 5da60cac753e94c423e5c98aaa5766ced5c961bb9aeb2cbd010262e9f7ee0098
+  url: https://github.com/quatrope/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e004dd6492b12532959df7ac8e0469c4a6f71d0f6dcf787c30d7ef23f7d72aba
+  patches:
```

```suggestion
  patches:   # [win]
```
If we apply patches only on win. Otherwise, ignore my comment